zones_test.go

```diff
@@ -273,6 +273,29 @@ func registerZoneMockResponder(testDomain string, zoneKind ZoneKind) {
 			return httpmock.NewStringResponse(http.StatusOK, makeDomainCanonical(testDomain)+"	3600	SOA	a.misconfigured.powerdns.server. hostmaster."+makeDomainCanonical(testDomain)+" 1 10800 3600 604800 3600"), nil
 		},
 	)
+
+	httpmock.RegisterResponder("PUT", generateTestAPIVHostURL()+"/zones/"+makeDomainCanonical(testDomain)+"/axfr-retrieve",
+		func(req *http.Request) (*http.Response, error) {
+			if res := verifyAPIKey(req); res != nil {
+				return res, nil
+			}
+
+			if req.Body != nil {
+				log.Print("Request body is not nil")
+				return httpmock.NewBytesResponse(http.StatusBadRequest, []byte{}), nil
+			}
+
+			acceptHeader := req.Header.Get("Accept")
+
+			if acceptHeader != "" && acceptHeader != "text/html" {
+				log.Print("Accept type must be text/html")
+				return httpmock.NewBytesResponse(http.StatusBadRequest, []byte{}), nil
+			}
+
```

Comment:
Is it really required to verify the `Accept` header? I cannot find any indication in the [API specification](https://github.com/PowerDNS/pdns/blob/master/docs/http-api/swagger/authoritative-api-swagger.yaml#L301).

Reply:
Unresolving this conversation. You accidentally removed the `Accept` header verification from the `/export` mock route, instead of the `/axfr-retrieve` route.